AdobeStockImageAdminUi/Test/Mftf/Test/UserSearchesStockImagesByKeywords.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="UserSearchesStockImagesByKeywords">
+        <annotations>
+            <features value="AdobeStockImagePanel"/>
+            <stories value="Access Adobe Stock Image panel via the Admin"/>
+            <title value="User searches stock images by keywords"/>
+            <description value="User searches stock images by keywords"/>
+            <severity value="CRITICAL"/>
+            <group value="AdobeStockIntegration"/>
+        </annotations>
+        <before>
+            <actionGroup ref="AdminAdobeStockSetConfigActionGroup" stepKey="setCorrectModuleConfig"/>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+        </before>
+        <after>
+            <actionGroup ref="logout" stepKey="logout"/>
+            <actionGroup ref="AdminAdobeStockSetConfigActionGroup" stepKey="cleanModuleConfig">
+                <argument name="enabled" value="0"/>
+                <argument name="apiKey" value=""/>
+                <argument name="privateKey" value=""/>
+            </actionGroup>
+        </after>
+        <actionGroup ref="AdminOpenAdobeStockSearchModalActionGroup" stepKey="openSearchModal"/>
+        <grabAttributeFrom selector="{{AdobeStockSection.firstImageAfterSearch}}" userInput="src"
+                           stepKey="getUrlFromFirstImageWithoutSearch"/>
+        <actionGroup ref="AdminSearchImagesOnModalActionGroup" stepKey="searchForPopular">
+            <argument name="query" value="popular"/>
+        </actionGroup>
+        <!-- Assert that User searches for "popular" using keyboard input and mouse navigation and sees "that grid was updated by new images"-->
+        <dontSeeElement selector="{{AdobeStockSection.imageSrc($getUrlFromFirstImageWithoutSearch)}}"
```

Comment:
what about a check that any item (or all 32 items) does exist on the page? Here we only check that the previous image is not here.

Reply:
@hws47a good idea, maybe i add action group that checks all grid if it contains images ?

Reply:
There is one in #239, it's expected to be merged to develop branch soon.

Reply:
did you mean this group ? **AdminAssertAdobeStockThumbnailsNumberActionGroup** 
however iv've implemented new one that chekcs total sount of images, can we use  this action group or we should use **AdminAssertAdobeStockThumbnailsNumberActionGroup** ?

Reply:
oh.. now i can see) your solution much better)